nilearn/decomposition/canica.py

```diff
@@ -123,7 +124,8 @@ def __init__(self, mask=None, n_components=20,
             mask=mask, memory=memory, memory_level=memory_level,
             n_jobs=n_jobs, verbose=verbose, do_cca=do_cca,
             n_components=n_components, smoothing_fwhm=smoothing_fwhm,
-            target_affine=target_affine, target_shape=target_shape)
+            target_affine=target_affine, target_shape=target_shape,
+            keep_data_mem=keep_data_mem)
```

Comment:
It is, given that otherwise we have to reload all data (filter_and_mask takes a considerable time)